dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/JobTask.kt

```diff
@@ -28,18 +28,26 @@ import com.spectralogic.dsbrowser.gui.services.Workers
 import com.spectralogic.dsbrowser.gui.services.jobinterruption.JobInterruptionStore
 import com.spectralogic.dsbrowser.gui.services.tasks.Ds3JobTask
 import com.spectralogic.dsbrowser.gui.util.ParseJobInterruptionMap
+import com.spectralogic.dsbrowser.util.Platform
```

Comment:
is this import used?

Reply:
InteliJ says it is, but removing it doesn't break anything.